internal/condition/condition_test.go

```diff
@@ -12,7 +13,7 @@ import (
 func testSetup(paramKey, paramValue string) (string, projectconfig.Module) {
 	bytes := make([]byte, 15)
 	_, _ = rand.Read(bytes)
-	name := string(bytes[:])
+	name := string(base64.StdEncoding.EncodeToString(bytes[:]))
```

Comment:
What was the issue here?

Reply:
on Mac the test errors out with bytes as filename

Reply:
:shrug: